micrometer/pom.xml

```diff
@@ -72,4 +85,22 @@
     </plugins>
   </build>
 
+  <repositories>
```

Comment:
yeah... this can't be

Reply:
I guess this temporary so that the PR can depend on Micrometer 1.10 (not yet GA).

Reply:
Yeah this PR would be merge once Micrometer 1.10.0 is out. Then there would be no SNAPSHOT dependencies.